Wikipedia/UI-V5/WMFHomeViewController.m

```diff
@@ -522,7 +522,10 @@ - (BOOL)tableView:(UITableView*)tableView shouldHighlightRowAtIndexPath:(NSIndex
 
 - (void)tableView:(UITableView*)tableView didSelectRowAtIndexPath:(NSIndexPath*)indexPath {
     id<WMFHomeSectionController> controller = [self sectionControllerForSectionAtIndex:indexPath.section];
-    MWKTitle* title                         = [controller titleForItemAtIndex:indexPath.row];
+    if ([controller respondsToSelector:@selector(shouldSelectItemAtIndex:)] && ![controller shouldSelectItemAtIndex:indexPath.item]) {
+        return;
```

Comment:
how is this possible? i thought you couldn't select items period unless the section had data